wordpress-mobile/WordPress-Android
# Stats traffic tab add traffic tab

Fixes #19941 

-----

| Before | After |
|--------|--------|
| ![Screenshot_20240115_163157](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/6bf58d4e-c7b8-42da-b99e-5f52546efd06) | ![Screenshot_20240115_163256](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/0758a72a-aced-4dbb-9438-78a62ccbcd0e) | 

## To Test:
Test 1:
- Enable `stats_traffic_tab` feature flag (Me -> Debug settings - Remote Features)
- Go to Stats
- `Verify` only Traffic tab, and Insights tabs are shown as in after column 
> [!NOTE] 
> Traffic tab is set to show Days tab for the time being since except for the first card with graph, the rest of the cards remain same.

Test 2:

- Disable feature flag
- `Verify` Stats tabs all show up as in before column, and work as before

-----

## Regression Notes

1. Potential unintended areas of impact

    - Tested various combinations

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Tested manually

3. What automated tests I added (or what prevented me from doing so)

    - Existing unit tests

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
